Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/PressableButtonInspector.cs

```diff
@@ -10,15 +10,46 @@ namespace Microsoft.MixedReality.Toolkit.Editor
     [CustomEditor(typeof(PressableButton))]
     public class PressableButtonInspector : UnityEditor.Editor
     {
-        private static bool editingEnabled = true;
+        // Struct use to store state of preview
+        // This lets us display accurate info while button is being pressed
+        private struct ButtonInfo
+        {
+            // Convenience fields for box collider info
+            public Bounds touchCageLocalBounds;
+            // Start pos for touch, also the origin of our cage
+            public Vector3 touchStartOrigin;
+            // Start and end pos for moving content
+            public float startPos;
+            public float endPos;
+            // Press, touch and release positions in z axis
+            public float pressDistPos;
+            public float touchStartPos;
+            public float releaseDistPos;
+            // Cage values
+            public float touchCageCenter;
+            public float touchCageSize;
+            // The actual values that the button uses
+            public float maxPushDistance;
+            public float pressDistance;
+            public float releaseDistanceDelta;
+        }
+
+        const string EditingEnabledKey = "MRTK_PressableButtonInspector_EditingEnabledKey";
+        const string VisiblePlanesKey = "MRTK_PressableButtonInspector_VisiblePlanesKey";
+        private static bool EditingEnabled = false;
+        private static bool VisiblePlanes = true;
+
         private const float labelMouseOverDistance = 0.025f;
 
         private static GUIStyle labelStyle;
 
         private PressableButton button;
         private Transform transform;
```

Comment:
Doesn't the name 'transform' override the monobehaviour 'transform'?

Reply:
This is an Editor script so no transform component exists.